packages/tables/docs/02-getting-started.md

```diff
@@ -308,6 +308,77 @@ In this example, clicking on each post will take you to the `posts.edit` route.
 
 If you'd like to [override the URL](columns#opening-urls) for a specific column, or instead [run a Livewire action](columns#running-actions) when a column is clicked, see the [columns documentation](columns#opening-urls).
 
+## Row appearance
+
+To change the appearance of rows conditionally on the record data, you may add the `getTableRecordClassesUsing()` method.
```

Comment:
```suggestion
You may want to conditionally style rows based on the record data. This can be achieved by specifying CSS classes to be applied to the row using the `getTableRecordClassesUsing()` method:
```